PyEIS/PyEIS.py

```diff
@@ -4698,7 +4698,7 @@ def EIS_plot(self, bode='off', fitting='off', rr='off', nyq_xlim='none', nyq_yli
         for i in range(len(self.df)):
             ax.plot(self.df[i].re, self.df[i].im, marker='o', ms=4, lw=2, color=colors[i], ls='-', label=self.label_cycleno[i])
             if fitting == 'on':
-                ax.plot(self.circuit_fit[i].real, -self.circuit_fit[i].imag, lw=0, marker='o', ms=8, mec='r', mew=1, mfc='none', label='')
+                ax.plot([i.real for i in self.circuit_fit[i]], [-i.imag for i in self.circuit_fit[i]], lw=0, marker='o', ms=8, mec='r', mew=1, mfc='none', label='')
```

Comment:
Could also use `self.circuit_fit[i].to_numpy().real`

Reply:
The same problem
`self.circuit_fit[i].to_numpy().real ` fixed it